Synthesis failed for nodejs-bigtable

Hello! Autosynth couldn't regenerate nodejs-bigtable. :broken_heart:

Here's the output from running `synth.py`:

```
2020-05-13 04:30:24,998 autosynth [INFO] > logs will be written to: /tmpfs/src/github/synthtool/logs/googleapis/nodejs-bigtable
2020-05-13 04:30:26,538 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:26,541 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:26,543 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:26,546 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:26,549 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:26,552 autosynth [DEBUG] > Running: {' '.join(command)}
Switched to branch 'autosynth'
2020-05-13 04:30:26,582 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:26,584 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:26,587 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:26,948 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,293 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,296 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,299 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,302 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,305 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,308 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,310 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,313 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,316 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,319 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,322 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,325 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,328 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,331 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,334 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,337 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,340 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,343 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,346 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,349 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,352 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,355 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,358 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,361 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,364 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,367 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,370 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,373 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,376 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,379 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,382 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,385 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,388 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,391 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,394 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,398 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,401 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,405 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,608 autosynth [DEBUG] > Running: {' '.join(command)}
Switched to branch 'autosynth-self'
2020-05-13 04:30:27,638 autosynth [DEBUG] > Running: {' '.join(command)}
Note: checking out '998dc4b5a41bcd7a1e12250623486cbe6ac9ab12'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-13 04:30:27,642 autosynth [DEBUG] > Running: {' '.join(command)}
Note: checking out 'a3a0bf0f6291d69f2ff3df7fcd63d28ee20ac727'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at a3a0bf0f BREAKING_CHANGE: Removing TimeSeriesQueryLanguageCondition as an alert condition type. The condition type is unsupported and unused. It was originally added for the Monitoring Query Language Alpha feature. refactor!: Drop support for TimeSeriesQueryLanguageCondition as an alert condition type.
2020-05-13 04:30:27,665 autosynth [DEBUG] > Running: {' '.join(command)}
Note: checking out 'be74d3e532faa47eb59f1a0eaebde0860d1d8ab4'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at be74d3e build: do not fail builds on codecov errors (#528)
2020-05-13 04:30:27,673 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,677 autosynth [DEBUG] > Running: {' '.join(command)}
Switched to branch 'autosynth-self-2'
2020-05-13 04:30:27,681 autosynth [INFO] > Running synthtool
2020-05-13 04:30:27,681 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-13 04:30:27,683 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:30:27,894 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py.
On branch autosynth-self-2
nothing to commit, working tree clean
2020-05-13 04:30:28,027 synthtool [DEBUG] > Ensuring dependencies.
DEBUG:synthtool:Ensuring dependencies.
2020-05-13 04:30:28,031 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2020-05-13 04:30:28,032 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-05-13 04:30:28,035 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:7bca0aa4c25f2cf6a07e1b6e1cd91a655c4cf010317b5b8fdde7e1aed0ba33d3
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-13 04:30:28,945 synthtool [DEBUG] > Generating code for: google/bigtable/v2.
DEBUG:synthtool:Generating code for: google/bigtable/v2.
2020-05-13 04:30:30,210 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpoeq4tczj.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpoeq4tczj.
2020-05-13 04:30:30,227 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:7bca0aa4c25f2cf6a07e1b6e1cd91a655c4cf010317b5b8fdde7e1aed0ba33d3
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-13 04:30:31,117 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.
DEBUG:synthtool:Generating code for: google/bigtable/admin/v2.
2020-05-13 04:30:32,497 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpxo1_hmha.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpxo1_hmha.
2020-05-13 04:30:32,530 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
2020-05-13 04:30:32,530 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
2020-05-13 04:30:32,530 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/publish.yml
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/lint.cfg
.kokoro/continuous/node10/samples-test.cfg
.kokoro/continuous/node10/system-test.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/continuous/node8/common.cfg
.kokoro/continuous/node8/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node10/docs.cfg
.kokoro/presubmit/node10/lint.cfg
.kokoro/presubmit/node10/samples-test.cfg
.kokoro/presubmit/node10/system-test.cfg
.kokoro/presubmit/node10/test.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/presubmit/node8/common.cfg
.kokoro/presubmit/node8/test.cfg
.kokoro/presubmit/windows/common.cfg
.kokoro/presubmit/windows/test.cfg
.kokoro/publish.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
renovate.json
samples/README.md
2020-05-13 04:30:32,674 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-05-13 04:30:32,674 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated

> protobufjs@6.9.0 postinstall /home/kbuilder/.cache/synthtool/nodejs-bigtable/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable
> npm run compile

npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> gts clean

version: 12
Removing build ...

> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> tsc -p . && cp -r proto* build/

[96msrc/instance.ts[0m:[93m890[0m:[93m20[0m - [91merror[0m[90m TS2339: [0mProperty 'gaxOptions' does not exist on type 'never'.
  The intersection 'GetTablesOptions & { parent: string; view: number; }' was reduced to 'never' because property 'view' has conflicting types in some constituents.

[7m890[0m     delete reqOpts.gaxOptions;
[7m   [0m [91m                   ~~~~~~~~~~[0m


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-13T11_31_02_989Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-13T11_31_03_038Z-debug.log
2020-05-13 04:31:03,066 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-05-13 04:31:03,067 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-05-13 04:31:03,105 autosynth [ERROR] > Synthesis failed
2020-05-13 04:31:03,105 autosynth [DEBUG] > Running: {' '.join(command)}
HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-13 04:31:03,119 autosynth [DEBUG] > Running: {' '.join(command)}
Switched to branch 'autosynth-self'
2020-05-13 04:31:03,127 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-05-13 04:31:03,128 autosynth [DEBUG] > Running: {' '.join(command)}
Note: checking out '998dc4b5a41bcd7a1e12250623486cbe6ac9ab12'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-13 04:31:03,136 autosynth [DEBUG] > Running: {' '.join(command)}
Previous HEAD position was a3a0bf0f BREAKING_CHANGE: Removing TimeSeriesQueryLanguageCondition as an alert condition type. The condition type is unsupported and unused. It was originally added for the Monitoring Query Language Alpha feature. refactor!: Drop support for TimeSeriesQueryLanguageCondition as an alert condition type.
HEAD is now at 89e89786 Publish Search Partner Data Ingestion logging proto for documentation
2020-05-13 04:31:03,160 autosynth [DEBUG] > Running: {' '.join(command)}
Previous HEAD position was be74d3e build: do not fail builds on codecov errors (#528)
HEAD is now at 84c4156 debug: wrap shell invocations in logging wrapper (#544)
2020-05-13 04:31:03,171 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:31:03,174 autosynth [DEBUG] > Running: {' '.join(command)}
Switched to branch 'autosynth-34'
2020-05-13 04:31:03,182 autosynth [INFO] > Running synthtool
2020-05-13 04:31:03,182 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-13 04:31:03,185 autosynth [DEBUG] > Running: {' '.join(command)}
2020-05-13 04:31:03,461 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py.
On branch autosynth-34
nothing to commit, working tree clean
2020-05-13 04:31:03,629 synthtool [DEBUG] > Ensuring dependencies.
DEBUG:synthtool:Ensuring dependencies.
2020-05-13 04:31:03,633 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2020-05-13 04:31:03,634 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-05-13 04:31:03,638 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:7bca0aa4c25f2cf6a07e1b6e1cd91a655c4cf010317b5b8fdde7e1aed0ba33d3
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-13 04:31:04,570 synthtool [DEBUG] > Generating code for: google/bigtable/v2.
DEBUG:synthtool:Generating code for: google/bigtable/v2.
2020-05-13 04:31:06,105 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp7jl1ug5c.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmp7jl1ug5c.
2020-05-13 04:31:06,121 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:7bca0aa4c25f2cf6a07e1b6e1cd91a655c4cf010317b5b8fdde7e1aed0ba33d3
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-13 04:31:07,011 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.
DEBUG:synthtool:Generating code for: google/bigtable/admin/v2.
2020-05-13 04:31:08,379 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpuj3a65l7.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpuj3a65l7.
2020-05-13 04:31:08,413 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
2020-05-13 04:31:08,413 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
2020-05-13 04:31:08,413 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/publish.yml
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/lint.cfg
.kokoro/continuous/node10/samples-test.cfg
.kokoro/continuous/node10/system-test.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/continuous/node8/common.cfg
.kokoro/continuous/node8/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node10/docs.cfg
.kokoro/presubmit/node10/lint.cfg
.kokoro/presubmit/node10/samples-test.cfg
.kokoro/presubmit/node10/system-test.cfg
.kokoro/presubmit/node10/test.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/presubmit/node8/common.cfg
.kokoro/presubmit/node8/test.cfg
.kokoro/presubmit/windows/common.cfg
.kokoro/presubmit/windows/test.cfg
.kokoro/publish.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
renovate.json
samples/README.md
2020-05-13 04:31:08,553 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-05-13 04:31:08,553 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable
> npm run compile

npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> gts clean

version: 12
Removing build ...

> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> tsc -p . && cp -r proto* build/

[96msrc/instance.ts[0m:[93m890[0m:[93m20[0m - [91merror[0m[90m TS2339: [0mProperty 'gaxOptions' does not exist on type 'never'.
  The intersection 'GetTablesOptions & { parent: string; view: number; }' was reduced to 'never' because property 'view' has conflicting types in some constituents.

[7m890[0m     delete reqOpts.gaxOptions;
[7m   [0m [91m                   ~~~~~~~~~~[0m


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-13T11_31_24_517Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-13T11_31_24_533Z-debug.log
2020-05-13 04:31:24,548 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-05-13 04:31:24,549 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-05-13 04:31:24,587 autosynth [ERROR] > Synthesis failed
2020-05-13 04:31:24,587 autosynth [DEBUG] > Running: {' '.join(command)}
HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-13 04:31:24,600 autosynth [DEBUG] > Running: {' '.join(command)}
Switched to branch 'autosynth'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 600, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 472, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 590, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 377, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 265, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](https://sponge/24b91e22-5723-4d3d-ad6a-192d8c1a7079).

Autosynth is still having trouble generating nodejs-bigtable. :sob:

Here's the output from running `synth.py`:

```
2020-05-14 05:21:25,261 autosynth [INFO] > logs will be written to: /tmpfs/src/github/synthtool/logs/googleapis/nodejs-bigtable
2020-05-14 05:21:26,246 autosynth [DEBUG] > Running: git config --global core.excludesfile /home/kbuilder/.autosynth-gitignore
2020-05-14 05:21:26,250 autosynth [DEBUG] > Running: git config user.name yoshi-automation
2020-05-14 05:21:26,254 autosynth [DEBUG] > Running: git config user.email yoshi-automation@google.com
2020-05-14 05:21:26,258 autosynth [DEBUG] > Running: git config push.default simple
2020-05-14 05:21:26,261 autosynth [DEBUG] > Running: git branch -f autosynth
2020-05-14 05:21:26,265 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-05-14 05:21:26,298 autosynth [DEBUG] > Running: git rev-parse --show-toplevel
2020-05-14 05:21:26,301 autosynth [DEBUG] > Running: git log -1 --pretty=%H
2020-05-14 05:21:26,305 autosynth [DEBUG] > Running: git remote get-url origin
2020-05-14 05:21:26,680 autosynth [DEBUG] > Running: git log a3a0bf0f6291d69f2ff3df7fcd63d28ee20ac727..HEAD --pretty=%H --no-decorate
2020-05-14 05:21:27,258 autosynth [DEBUG] > Running: git log be74d3e532faa47eb59f1a0eaebde0860d1d8ab4..HEAD --pretty=%H --no-decorate
2020-05-14 05:21:27,261 autosynth [DEBUG] > Running: git log -1 --pretty=%at a3a0bf0f6291d69f2ff3df7fcd63d28ee20ac727
2020-05-14 05:21:27,264 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6dfd72d028a0d0a43764e060f7b15e004385c3a1
2020-05-14 05:21:27,267 autosynth [DEBUG] > Running: git log -1 --pretty=%at 839fae42335ee1bb1e70767f3e6c51738683892b
2020-05-14 05:21:27,270 autosynth [DEBUG] > Running: git log -1 --pretty=%at 30cfca094376e4904e32e71c838a81169fd4a2e2
2020-05-14 05:21:27,274 autosynth [DEBUG] > Running: git log -1 --pretty=%at c08dcec05ce1c181bcdbce59cabba36e0e541ff6
2020-05-14 05:21:27,277 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2fc2caaacb15949c7f80426bfc7dafdd41dbc333
2020-05-14 05:21:27,280 autosynth [DEBUG] > Running: git log -1 --pretty=%at 684dfea7decfeca7a7526ea96a8e9256694dd5d8
2020-05-14 05:21:27,283 autosynth [DEBUG] > Running: git log -1 --pretty=%at fb8f62b6784f43faf4b64179c57ce4b4931b1a00
2020-05-14 05:21:27,286 autosynth [DEBUG] > Running: git log -1 --pretty=%at d3cc7bbf41afdfd6677cc1ab0d53965625cc073b
2020-05-14 05:21:27,289 autosynth [DEBUG] > Running: git log -1 --pretty=%at 83816bb3093686a28af2891db5b7506614a820b1
2020-05-14 05:21:27,292 autosynth [DEBUG] > Running: git log -1 --pretty=%at 73d4b5d9a791f8b1ee63d439ffe909bb8ffa07f7
2020-05-14 05:21:27,295 autosynth [DEBUG] > Running: git log -1 --pretty=%at 482e5206e05e4ba8cd79738f6fb7a521c8c23555
2020-05-14 05:21:27,298 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1019b0d9dbd999e1107f90247fcc478678d1105c
2020-05-14 05:21:27,301 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4186d3e5424edc088a0f4dbce78f02dd860f3de8
2020-05-14 05:21:27,304 autosynth [DEBUG] > Running: git log -1 --pretty=%at aed11c01e52921613b9ee469c2d85f5f33175fb7
2020-05-14 05:21:27,308 autosynth [DEBUG] > Running: git log -1 --pretty=%at edd3b80fb770548d6ad780105f1782de6ff73ea0
2020-05-14 05:21:27,311 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1b8b77db10a0bd0ff47a4259ae9804b6237bc1c7
2020-05-14 05:21:27,315 autosynth [DEBUG] > Running: git log -1 --pretty=%at bf17ae5fd93929beb44ac4c6b04f5088c3ee4a02
2020-05-14 05:21:27,318 autosynth [DEBUG] > Running: git log -1 --pretty=%at 358c09c45c681c6773b49aeb136f0a7ccbf7720c
2020-05-14 05:21:27,321 autosynth [DEBUG] > Running: git log -1 --pretty=%at c8e291e6a4d60771219205b653715d5aeec3e96b
2020-05-14 05:21:27,325 autosynth [DEBUG] > Running: git log -1 --pretty=%at 89e89786896d256c70f43e68a975470c4f4f220e
2020-05-14 05:21:27,328 autosynth [DEBUG] > Running: git log -1 --pretty=%at 57253ab4c0a78d2351cff0aa638ff81b89b19afc
2020-05-14 05:21:27,331 autosynth [DEBUG] > Running: git log -1 --pretty=%at be74d3e532faa47eb59f1a0eaebde0860d1d8ab4
2020-05-14 05:21:27,334 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4674113712c0c7ada19e6a8219d7963ff174b392
2020-05-14 05:21:27,337 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5bbfd095faedfe273819d266f21e402192a29041
2020-05-14 05:21:27,340 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4fa923bd3dafb91df8613accbe2230299cc5b98e
2020-05-14 05:21:27,344 autosynth [DEBUG] > Running: git log -1 --pretty=%at 55cdc844877d97139f25004229842624a6a86a02
2020-05-14 05:21:27,347 autosynth [DEBUG] > Running: git log -1 --pretty=%at 98c50772ec23295c64cf0d2ddf199ea52961fd4c
2020-05-14 05:21:27,350 autosynth [DEBUG] > Running: git log -1 --pretty=%at ba909fca409f6b38eae0fa735614e127d1fc0deb
2020-05-14 05:21:27,353 autosynth [DEBUG] > Running: git log -1 --pretty=%at 7482e79a82e353248769d819788adc1213e8c207
2020-05-14 05:21:27,356 autosynth [DEBUG] > Running: git log -1 --pretty=%at a7759f81c25396207d46532ed389ad4d34879857
2020-05-14 05:21:27,359 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5b48b0716a36ca069db3038da7e205c87a22ed19
2020-05-14 05:21:27,362 autosynth [DEBUG] > Running: git log -1 --pretty=%at c585ac3b5eff5cd2097a5315ffd9cf4823cc1ed2
2020-05-14 05:21:27,365 autosynth [DEBUG] > Running: git log -1 --pretty=%at b0461724be19443075b08c10d4a345cb217002b5
2020-05-14 05:21:27,368 autosynth [DEBUG] > Running: git log -1 --pretty=%at 84c4156c49be9dcabacc8fd7b0585b6fd789ae47
2020-05-14 05:21:27,371 autosynth [DEBUG] > Running: git log -1 --pretty=%at f503622985e230a6792730bbc3b7746c11fce09e
2020-05-14 05:21:27,374 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3d2a7d0e21387ed455c966da9f9897b0a4bc5bb8
2020-05-14 05:21:27,377 autosynth [DEBUG] > Running: git log -1 --pretty=%at 7b7f386b393947a542b87707499f4458136f4f61
2020-05-14 05:21:27,381 autosynth [DEBUG] > Running: git branch -f autosynth-self
2020-05-14 05:21:27,385 autosynth [DEBUG] > Running: git branch -f autosynth-googleapis
2020-05-14 05:21:27,388 autosynth [DEBUG] > Running: git branch -f autosynth-synthtool
2020-05-14 05:21:27,577 autosynth [DEBUG] > Running: git checkout autosynth-self
Switched to branch 'autosynth-self'
2020-05-14 05:21:27,608 autosynth [DEBUG] > Running: git checkout 998dc4b5a41bcd7a1e12250623486cbe6ac9ab12
Note: checking out '998dc4b5a41bcd7a1e12250623486cbe6ac9ab12'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-14 05:21:27,612 autosynth [DEBUG] > Running: git checkout a3a0bf0f6291d69f2ff3df7fcd63d28ee20ac727
Note: checking out 'a3a0bf0f6291d69f2ff3df7fcd63d28ee20ac727'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at a3a0bf0f BREAKING_CHANGE: Removing TimeSeriesQueryLanguageCondition as an alert condition type. The condition type is unsupported and unused. It was originally added for the Monitoring Query Language Alpha feature. refactor!: Drop support for TimeSeriesQueryLanguageCondition as an alert condition type.
2020-05-14 05:21:27,636 autosynth [DEBUG] > Running: git checkout be74d3e532faa47eb59f1a0eaebde0860d1d8ab4
Note: checking out 'be74d3e532faa47eb59f1a0eaebde0860d1d8ab4'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at be74d3e build: do not fail builds on codecov errors (#528)
2020-05-14 05:21:27,645 autosynth [DEBUG] > Running: git branch -f autosynth-self-2
2020-05-14 05:21:27,648 autosynth [DEBUG] > Running: git checkout autosynth-self-2
Switched to branch 'autosynth-self-2'
2020-05-14 05:21:27,653 autosynth [INFO] > Running synthtool
2020-05-14 05:21:27,653 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-14 05:21:27,655 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-05-14 05:21:27,870 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py.
On branch autosynth-self-2
nothing to commit, working tree clean
2020-05-14 05:21:28,004 synthtool [DEBUG] > Ensuring dependencies.
DEBUG:synthtool:Ensuring dependencies.
2020-05-14 05:21:28,009 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2020-05-14 05:21:28,010 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-05-14 05:21:28,013 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:7bca0aa4c25f2cf6a07e1b6e1cd91a655c4cf010317b5b8fdde7e1aed0ba33d3
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-14 05:21:28,928 synthtool [DEBUG] > Generating code for: google/bigtable/v2.
DEBUG:synthtool:Generating code for: google/bigtable/v2.
2020-05-14 05:21:30,268 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmphr8enxii.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmphr8enxii.
2020-05-14 05:21:30,284 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:7bca0aa4c25f2cf6a07e1b6e1cd91a655c4cf010317b5b8fdde7e1aed0ba33d3
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-14 05:21:31,210 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.
DEBUG:synthtool:Generating code for: google/bigtable/admin/v2.
2020-05-14 05:21:32,678 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp328agxzv.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmp328agxzv.
2020-05-14 05:21:32,711 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
2020-05-14 05:21:32,711 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
2020-05-14 05:21:32,711 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/publish.yml
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/lint.cfg
.kokoro/continuous/node10/samples-test.cfg
.kokoro/continuous/node10/system-test.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/continuous/node8/common.cfg
.kokoro/continuous/node8/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node10/docs.cfg
.kokoro/presubmit/node10/lint.cfg
.kokoro/presubmit/node10/samples-test.cfg
.kokoro/presubmit/node10/system-test.cfg
.kokoro/presubmit/node10/test.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/presubmit/node8/common.cfg
.kokoro/presubmit/node8/test.cfg
.kokoro/presubmit/windows/common.cfg
.kokoro/presubmit/windows/test.cfg
.kokoro/publish.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
renovate.json
samples/README.md
2020-05-14 05:21:32,865 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-05-14 05:21:32,865 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated

> protobufjs@6.9.0 postinstall /home/kbuilder/.cache/synthtool/nodejs-bigtable/node_modules/protobufjs
> node scripts/postinstall


> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable
> npm run compile

npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> gts clean

version: 12
Removing build ...

> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> tsc -p . && cp -r proto* build/

[96msrc/instance.ts[0m:[93m890[0m:[93m20[0m - [91merror[0m[90m TS2339: [0mProperty 'gaxOptions' does not exist on type 'never'.
  The intersection 'GetTablesOptions & { parent: string; view: number; }' was reduced to 'never' because property 'view' has conflicting types in some constituents.

[7m890[0m     delete reqOpts.gaxOptions;
[7m   [0m [91m                   ~~~~~~~~~~[0m


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-14T12_22_05_602Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-14T12_22_05_641Z-debug.log
2020-05-14 05:22:05,671 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-05-14 05:22:05,673 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-05-14 05:22:05,716 autosynth [ERROR] > Synthesis failed
2020-05-14 05:22:05,716 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-14 05:22:05,731 autosynth [DEBUG] > Running: git checkout autosynth-self
Switched to branch 'autosynth-self'
2020-05-14 05:22:05,741 autosynth [ERROR] > Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.
2020-05-14 05:22:05,741 autosynth [DEBUG] > Running: git checkout 998dc4b5a41bcd7a1e12250623486cbe6ac9ab12
Note: checking out '998dc4b5a41bcd7a1e12250623486cbe6ac9ab12'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-14 05:22:05,750 autosynth [DEBUG] > Running: git checkout 57253ab4c0a78d2351cff0aa638ff81b89b19afc
Previous HEAD position was a3a0bf0f BREAKING_CHANGE: Removing TimeSeriesQueryLanguageCondition as an alert condition type. The condition type is unsupported and unused. It was originally added for the Monitoring Query Language Alpha feature. refactor!: Drop support for TimeSeriesQueryLanguageCondition as an alert condition type.
HEAD is now at 57253ab4 chore: delete gapic config for oslogin/v1beta1 API
2020-05-14 05:22:05,777 autosynth [DEBUG] > Running: git checkout 7b7f386b393947a542b87707499f4458136f4f61
Previous HEAD position was be74d3e build: do not fail builds on codecov errors (#528)
HEAD is now at 7b7f386 fix: use env for token (#550)
2020-05-14 05:22:05,789 autosynth [DEBUG] > Running: git branch -f autosynth-38
2020-05-14 05:22:05,793 autosynth [DEBUG] > Running: git checkout autosynth-38
Switched to branch 'autosynth-38'
2020-05-14 05:22:05,802 autosynth [INFO] > Running synthtool
2020-05-14 05:22:05,802 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-05-14 05:22:05,804 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-05-14 05:22:06,024 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py.
On branch autosynth-38
nothing to commit, working tree clean
2020-05-14 05:22:06,170 synthtool [DEBUG] > Ensuring dependencies.
DEBUG:synthtool:Ensuring dependencies.
2020-05-14 05:22:06,175 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2020-05-14 05:22:06,176 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/googleapis
2020-05-14 05:22:06,180 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:7bca0aa4c25f2cf6a07e1b6e1cd91a655c4cf010317b5b8fdde7e1aed0ba33d3
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-14 05:22:07,121 synthtool [DEBUG] > Generating code for: google/bigtable/v2.
DEBUG:synthtool:Generating code for: google/bigtable/v2.
2020-05-14 05:22:08,631 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp_12c6pkf.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmp_12c6pkf.
2020-05-14 05:22:08,647 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:7bca0aa4c25f2cf6a07e1b6e1cd91a655c4cf010317b5b8fdde7e1aed0ba33d3
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-14 05:22:09,582 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.
DEBUG:synthtool:Generating code for: google/bigtable/admin/v2.
2020-05-14 05:22:11,016 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpm69uhooh.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpm69uhooh.
2020-05-14 05:22:11,049 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
2020-05-14 05:22:11,049 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
2020-05-14 05:22:11,049 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/publish.yml
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/lint.cfg
.kokoro/continuous/node10/samples-test.cfg
.kokoro/continuous/node10/system-test.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/continuous/node8/common.cfg
.kokoro/continuous/node8/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node10/docs.cfg
.kokoro/presubmit/node10/lint.cfg
.kokoro/presubmit/node10/samples-test.cfg
.kokoro/presubmit/node10/system-test.cfg
.kokoro/presubmit/node10/test.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/presubmit/node8/common.cfg
.kokoro/presubmit/node8/test.cfg
.kokoro/presubmit/windows/common.cfg
.kokoro/presubmit/windows/test.cfg
.kokoro/publish.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
renovate.json
samples/README.md
2020-05-14 05:22:11,197 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-05-14 05:22:11,197 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable
> npm run compile

npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> gts clean

version: 12
Removing build ...

> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> tsc -p . && cp -r proto* build/

[96msrc/instance.ts[0m:[93m890[0m:[93m20[0m - [91merror[0m[90m TS2339: [0mProperty 'gaxOptions' does not exist on type 'never'.
  The intersection 'GetTablesOptions & { parent: string; view: number; }' was reduced to 'never' because property 'view' has conflicting types in some constituents.

[7m890[0m     delete reqOpts.gaxOptions;
[7m   [0m [91m                   ~~~~~~~~~~[0m


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-14T12_22_28_061Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-14T12_22_28_079Z-debug.log
2020-05-14 05:22:28,095 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-05-14 05:22:28,097 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-05-14 05:22:28,140 autosynth [ERROR] > Synthesis failed
2020-05-14 05:22:28,140 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-14 05:22:28,154 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 600, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 472, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 590, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 377, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 265, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](https://sponge/ea51e8ba-def4-460f-9acd-a09a559fce5d).

Autosynth is still having trouble generating nodejs-bigtable. :sob:

Here's the output from running `synth.py`:

```
us: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-14 15:01:56,755 synthtool [DEBUG] > Generating code for: google/bigtable/v2.
DEBUG:synthtool:Generating code for: google/bigtable/v2.
2020-05-14 15:01:58,044 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpzcm1sren.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpzcm1sren.
2020-05-14 15:01:58,059 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:7bca0aa4c25f2cf6a07e1b6e1cd91a655c4cf010317b5b8fdde7e1aed0ba33d3
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-14 15:01:58,967 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.
DEBUG:synthtool:Generating code for: google/bigtable/admin/v2.
2020-05-14 15:02:00,289 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpu05zhyhx.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpu05zhyhx.
2020-05-14 15:02:00,317 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
2020-05-14 15:02:00,318 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
2020-05-14 15:02:00,318 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/publish.yml
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/lint.cfg
.kokoro/continuous/node10/samples-test.cfg
.kokoro/continuous/node10/system-test.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/continuous/node8/common.cfg
.kokoro/continuous/node8/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node10/docs.cfg
.kokoro/presubmit/node10/lint.cfg
.kokoro/presubmit/node10/samples-test.cfg
.kokoro/presubmit/node10/system-test.cfg
.kokoro/presubmit/node10/test.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/presubmit/node8/common.cfg
.kokoro/presubmit/node8/test.cfg
.kokoro/presubmit/windows/common.cfg
.kokoro/presubmit/windows/test.cfg
.kokoro/publish.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
renovate.json
samples/README.md
2020-05-14 15:02:00,447 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-05-14 15:02:00,447 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable
> npm run compile

npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> gts clean

version: 12
Removing build ...

> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> tsc -p . && cp -r proto* build/

[96msrc/instance.ts[0m:[93m890[0m:[93m20[0m - [91merror[0m[90m TS2339: [0mProperty 'gaxOptions' does not exist on type 'never'.
  The intersection 'GetTablesOptions & { parent: string; view: number; }' was reduced to 'never' because property 'view' has conflicting types in some constituents.

[7m890[0m     delete reqOpts.gaxOptions;
[7m   [0m [91m                   ~~~~~~~~~~[0m


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-14T22_02_15_128Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-14T22_02_15_148Z-debug.log
2020-05-14 15:02:15,163 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-05-14 15:02:15,164 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-05-14 15:02:15,216 autosynth [ERROR] > Synthesis failed
2020-05-14 15:02:15,216 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-14 15:02:15,229 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 600, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 472, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 590, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 377, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 265, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](https://sponge/30e633c5-055b-437c-9be8-8b2a9ed80d0d).

Autosynth is still having trouble generating nodejs-bigtable. :sob:

Here's the output from running `synth.py`:

```
us: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-14 18:23:03,288 synthtool [DEBUG] > Generating code for: google/bigtable/v2.
DEBUG:synthtool:Generating code for: google/bigtable/v2.
2020-05-14 18:23:04,871 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpzop0f3oe.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpzop0f3oe.
2020-05-14 18:23:04,885 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:7bca0aa4c25f2cf6a07e1b6e1cd91a655c4cf010317b5b8fdde7e1aed0ba33d3
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-14 18:23:05,822 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.
DEBUG:synthtool:Generating code for: google/bigtable/admin/v2.
2020-05-14 18:23:07,144 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpecwku_b1.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpecwku_b1.
2020-05-14 18:23:07,172 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
2020-05-14 18:23:07,173 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
2020-05-14 18:23:07,173 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/publish.yml
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/lint.cfg
.kokoro/continuous/node10/samples-test.cfg
.kokoro/continuous/node10/system-test.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/continuous/node8/common.cfg
.kokoro/continuous/node8/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node10/docs.cfg
.kokoro/presubmit/node10/lint.cfg
.kokoro/presubmit/node10/samples-test.cfg
.kokoro/presubmit/node10/system-test.cfg
.kokoro/presubmit/node10/test.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/presubmit/node8/common.cfg
.kokoro/presubmit/node8/test.cfg
.kokoro/presubmit/windows/common.cfg
.kokoro/presubmit/windows/test.cfg
.kokoro/publish.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
renovate.json
samples/README.md
2020-05-14 18:23:07,304 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-05-14 18:23:07,305 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable
> npm run compile

npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> gts clean

version: 12
Removing build ...

> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> tsc -p . && cp -r proto* build/

[96msrc/instance.ts[0m:[93m890[0m:[93m20[0m - [91merror[0m[90m TS2339: [0mProperty 'gaxOptions' does not exist on type 'never'.
  The intersection 'GetTablesOptions & { parent: string; view: number; }' was reduced to 'never' because property 'view' has conflicting types in some constituents.

[7m890[0m     delete reqOpts.gaxOptions;
[7m   [0m [91m                   ~~~~~~~~~~[0m


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-15T01_23_22_242Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-15T01_23_22_263Z-debug.log
2020-05-14 18:23:22,279 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-05-14 18:23:22,280 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-05-14 18:23:22,332 autosynth [ERROR] > Synthesis failed
2020-05-14 18:23:22,332 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-14 18:23:22,346 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 601, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 473, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 591, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 368, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 378, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](https://sponge/ab56f2e1-d6ee-423b-92fe-ef8544e78c16).

Autosynth is still having trouble generating nodejs-bigtable. :sob:

Here's the output from running `synth.py`:

```
us: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-15 04:31:22,910 synthtool [DEBUG] > Generating code for: google/bigtable/v2.
DEBUG:synthtool:Generating code for: google/bigtable/v2.
2020-05-15 04:31:24,451 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp28bbkugm.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmp28bbkugm.
2020-05-15 04:31:24,467 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:7bca0aa4c25f2cf6a07e1b6e1cd91a655c4cf010317b5b8fdde7e1aed0ba33d3
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-15 04:31:25,377 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.
DEBUG:synthtool:Generating code for: google/bigtable/admin/v2.
2020-05-15 04:31:26,728 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp8s1mdym6.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmp8s1mdym6.
2020-05-15 04:31:26,760 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
2020-05-15 04:31:26,761 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
2020-05-15 04:31:26,761 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/publish.yml
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/lint.cfg
.kokoro/continuous/node10/samples-test.cfg
.kokoro/continuous/node10/system-test.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/continuous/node8/common.cfg
.kokoro/continuous/node8/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node10/docs.cfg
.kokoro/presubmit/node10/lint.cfg
.kokoro/presubmit/node10/samples-test.cfg
.kokoro/presubmit/node10/system-test.cfg
.kokoro/presubmit/node10/test.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/presubmit/node8/common.cfg
.kokoro/presubmit/node8/test.cfg
.kokoro/presubmit/windows/common.cfg
.kokoro/presubmit/windows/test.cfg
.kokoro/publish.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
renovate.json
samples/README.md
2020-05-15 04:31:26,905 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-05-15 04:31:26,905 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable
> npm run compile

npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> gts clean

version: 12
Removing build ...

> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> tsc -p . && cp -r proto* build/

[96msrc/instance.ts[0m:[93m890[0m:[93m20[0m - [91merror[0m[90m TS2339: [0mProperty 'gaxOptions' does not exist on type 'never'.
  The intersection 'GetTablesOptions & { parent: string; view: number; }' was reduced to 'never' because property 'view' has conflicting types in some constituents.

[7m890[0m     delete reqOpts.gaxOptions;
[7m   [0m [91m                   ~~~~~~~~~~[0m


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-15T11_31_42_916Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-15T11_31_42_934Z-debug.log
2020-05-15 04:31:42,949 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-05-15 04:31:42,950 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-05-15 04:31:42,988 autosynth [ERROR] > Synthesis failed
2020-05-15 04:31:42,988 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-15 04:31:43,001 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 601, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 473, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 591, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 368, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 378, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](https://sponge/9005fcf3-dd82-4e8a-9eed-ade452c70736).

Autosynth is still having trouble generating nodejs-bigtable. :sob:

Here's the output from running `synth.py`:

```
ol [DEBUG] > Generating code for: google/bigtable/v2.
DEBUG:synthtool:Generating code for: google/bigtable/v2.
2020-05-15 11:52:56,576 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmprbinpvi2.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmprbinpvi2.
2020-05-15 11:52:56,593 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:7bca0aa4c25f2cf6a07e1b6e1cd91a655c4cf010317b5b8fdde7e1aed0ba33d3
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-15 11:52:57,488 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.
DEBUG:synthtool:Generating code for: google/bigtable/admin/v2.
2020-05-15 11:52:58,768 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpyd82q9ow.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpyd82q9ow.
2020-05-15 11:52:58,800 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
2020-05-15 11:52:58,800 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
2020-05-15 11:52:58,800 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/publish.yml
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/lint.cfg
.kokoro/continuous/node10/samples-test.cfg
.kokoro/continuous/node10/system-test.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/continuous/node8/common.cfg
.kokoro/continuous/node8/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node10/docs.cfg
.kokoro/presubmit/node10/lint.cfg
.kokoro/presubmit/node10/samples-test.cfg
.kokoro/presubmit/node10/system-test.cfg
.kokoro/presubmit/node10/test.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/presubmit/node8/common.cfg
.kokoro/presubmit/node8/test.cfg
.kokoro/presubmit/windows/common.cfg
.kokoro/presubmit/windows/test.cfg
.kokoro/publish.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
renovate.json
samples/README.md
2020-05-15 11:52:58,938 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-05-15 11:52:58,938 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable
> npm run compile

npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> gts clean

version: 12
Removing build ...

> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> tsc -p . && cp -r proto* build/

[96msrc/instance.ts[0m:[93m890[0m:[93m20[0m - [91merror[0m[90m TS2339: [0mProperty 'gaxOptions' does not exist on type 'never'.
  The intersection 'GetTablesOptions & { parent: string; view: number; }' was reduced to 'never' because property 'view' has conflicting types in some constituents.

[7m890[0m     delete reqOpts.gaxOptions;
[7m   [0m [91m                   ~~~~~~~~~~[0m


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-15T18_53_14_640Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-15T18_53_14_661Z-debug.log
2020-05-15 11:53:14,674 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-05-15 11:53:14,676 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-05-15 11:53:14,712 autosynth [ERROR] > Synthesis failed
2020-05-15 11:53:14,712 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-15 11:53:14,726 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-05-15 11:53:14,734 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 612, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 473, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 592, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 368, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 378, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge/3ec8a997-6287-4049-9d29-bcba78de1be3).

Autosynth is still having trouble generating nodejs-bigtable. :sob:

Here's the output from running `synth.py`:

```
ol [DEBUG] > Generating code for: google/bigtable/v2.
DEBUG:synthtool:Generating code for: google/bigtable/v2.
2020-05-16 04:38:57,089 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp91uqz_jp.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmp91uqz_jp.
2020-05-16 04:38:57,107 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:7bca0aa4c25f2cf6a07e1b6e1cd91a655c4cf010317b5b8fdde7e1aed0ba33d3
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-16 04:38:58,006 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.
DEBUG:synthtool:Generating code for: google/bigtable/admin/v2.
2020-05-16 04:38:59,361 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpsahm7obd.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpsahm7obd.
2020-05-16 04:38:59,393 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
2020-05-16 04:38:59,394 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
2020-05-16 04:38:59,394 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/publish.yml
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/lint.cfg
.kokoro/continuous/node10/samples-test.cfg
.kokoro/continuous/node10/system-test.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/continuous/node8/common.cfg
.kokoro/continuous/node8/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node10/docs.cfg
.kokoro/presubmit/node10/lint.cfg
.kokoro/presubmit/node10/samples-test.cfg
.kokoro/presubmit/node10/system-test.cfg
.kokoro/presubmit/node10/test.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/presubmit/node8/common.cfg
.kokoro/presubmit/node8/test.cfg
.kokoro/presubmit/windows/common.cfg
.kokoro/presubmit/windows/test.cfg
.kokoro/publish.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
renovate.json
samples/README.md
2020-05-16 04:38:59,534 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-05-16 04:38:59,535 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable
> npm run compile

npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> gts clean

version: 12
Removing build ...

> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> tsc -p . && cp -r proto* build/

[96msrc/instance.ts[0m:[93m890[0m:[93m20[0m - [91merror[0m[90m TS2339: [0mProperty 'gaxOptions' does not exist on type 'never'.
  The intersection 'GetTablesOptions & { parent: string; view: number; }' was reduced to 'never' because property 'view' has conflicting types in some constituents.

[7m890[0m     delete reqOpts.gaxOptions;
[7m   [0m [91m                   ~~~~~~~~~~[0m


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-16T11_39_15_280Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-16T11_39_15_300Z-debug.log
2020-05-16 04:39:15,314 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-05-16 04:39:15,315 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-05-16 04:39:15,353 autosynth [ERROR] > Synthesis failed
2020-05-16 04:39:15,353 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-16 04:39:15,367 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-05-16 04:39:15,375 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge/74eabfaf-a964-4a5b-b26c-c28d4ae22df0).

Autosynth is still having trouble generating nodejs-bigtable. :sob:

Here's the output from running `synth.py`:

```
ol [DEBUG] > Generating code for: google/bigtable/v2.
DEBUG:synthtool:Generating code for: google/bigtable/v2.
2020-05-17 04:36:33,269 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp_bm9kg00.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmp_bm9kg00.
2020-05-17 04:36:33,285 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:7bca0aa4c25f2cf6a07e1b6e1cd91a655c4cf010317b5b8fdde7e1aed0ba33d3
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-17 04:36:34,234 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.
DEBUG:synthtool:Generating code for: google/bigtable/admin/v2.
2020-05-17 04:36:35,613 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpwd2_d6p2.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpwd2_d6p2.
2020-05-17 04:36:35,646 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
2020-05-17 04:36:35,646 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
2020-05-17 04:36:35,646 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/publish.yml
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/lint.cfg
.kokoro/continuous/node10/samples-test.cfg
.kokoro/continuous/node10/system-test.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/continuous/node8/common.cfg
.kokoro/continuous/node8/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node10/docs.cfg
.kokoro/presubmit/node10/lint.cfg
.kokoro/presubmit/node10/samples-test.cfg
.kokoro/presubmit/node10/system-test.cfg
.kokoro/presubmit/node10/test.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/presubmit/node8/common.cfg
.kokoro/presubmit/node8/test.cfg
.kokoro/presubmit/windows/common.cfg
.kokoro/presubmit/windows/test.cfg
.kokoro/publish.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
renovate.json
samples/README.md
2020-05-17 04:36:35,789 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-05-17 04:36:35,790 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable
> npm run compile

npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> gts clean

version: 12
Removing build ...

> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> tsc -p . && cp -r proto* build/

[96msrc/instance.ts[0m:[93m890[0m:[93m20[0m - [91merror[0m[90m TS2339: [0mProperty 'gaxOptions' does not exist on type 'never'.
  The intersection 'GetTablesOptions & { parent: string; view: number; }' was reduced to 'never' because property 'view' has conflicting types in some constituents.

[7m890[0m     delete reqOpts.gaxOptions;
[7m   [0m [91m                   ~~~~~~~~~~[0m


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-17T11_36_51_855Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-17T11_36_51_873Z-debug.log
2020-05-17 04:36:51,887 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-05-17 04:36:51,888 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-05-17 04:36:51,930 autosynth [ERROR] > Synthesis failed
2020-05-17 04:36:51,930 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-17 04:36:51,943 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-05-17 04:36:51,952 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge/f108f3b1-7c85-4ba3-9ecd-c042b18b604a).

Autosynth is still having trouble generating nodejs-bigtable. :sob:

Here's the output from running `synth.py`:

```
ol [DEBUG] > Generating code for: google/bigtable/v2.
DEBUG:synthtool:Generating code for: google/bigtable/v2.
2020-05-18 04:33:26,886 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpsf9cfk0g.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpsf9cfk0g.
2020-05-18 04:33:26,900 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:7bca0aa4c25f2cf6a07e1b6e1cd91a655c4cf010317b5b8fdde7e1aed0ba33d3
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-18 04:33:27,811 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.
DEBUG:synthtool:Generating code for: google/bigtable/admin/v2.
2020-05-18 04:33:29,158 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp0kke8y0l.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmp0kke8y0l.
2020-05-18 04:33:29,190 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
2020-05-18 04:33:29,190 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
2020-05-18 04:33:29,190 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/publish.yml
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/lint.cfg
.kokoro/continuous/node10/samples-test.cfg
.kokoro/continuous/node10/system-test.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/continuous/node8/common.cfg
.kokoro/continuous/node8/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node10/docs.cfg
.kokoro/presubmit/node10/lint.cfg
.kokoro/presubmit/node10/samples-test.cfg
.kokoro/presubmit/node10/system-test.cfg
.kokoro/presubmit/node10/test.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/presubmit/node8/common.cfg
.kokoro/presubmit/node8/test.cfg
.kokoro/presubmit/windows/common.cfg
.kokoro/presubmit/windows/test.cfg
.kokoro/publish.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
renovate.json
samples/README.md
2020-05-18 04:33:29,318 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-05-18 04:33:29,318 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable
> npm run compile

npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> gts clean

version: 12
Removing build ...

> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> tsc -p . && cp -r proto* build/

[96msrc/instance.ts[0m:[93m890[0m:[93m20[0m - [91merror[0m[90m TS2339: [0mProperty 'gaxOptions' does not exist on type 'never'.
  The intersection 'GetTablesOptions & { parent: string; view: number; }' was reduced to 'never' because property 'view' has conflicting types in some constituents.

[7m890[0m     delete reqOpts.gaxOptions;
[7m   [0m [91m                   ~~~~~~~~~~[0m


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-18T11_33_44_045Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-18T11_33_44_070Z-debug.log
2020-05-18 04:33:44,086 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-05-18 04:33:44,087 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-05-18 04:33:44,139 autosynth [ERROR] > Synthesis failed
2020-05-18 04:33:44,139 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-18 04:33:44,153 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-05-18 04:33:44,164 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge/ba06c5be-cfd9-437c-88f8-032a50b1776e).

Autosynth is still having trouble generating nodejs-bigtable. :sob:

Here's the output from running `synth.py`:

```
ol [DEBUG] > Generating code for: google/bigtable/v2.
DEBUG:synthtool:Generating code for: google/bigtable/v2.
2020-05-19 04:35:22,731 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmppf0_2p1j.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmppf0_2p1j.
2020-05-19 04:35:22,745 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:7bca0aa4c25f2cf6a07e1b6e1cd91a655c4cf010317b5b8fdde7e1aed0ba33d3
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-19 04:35:23,640 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.
DEBUG:synthtool:Generating code for: google/bigtable/admin/v2.
2020-05-19 04:35:24,951 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpes6rrj6p.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpes6rrj6p.
2020-05-19 04:35:24,980 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
2020-05-19 04:35:24,981 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
2020-05-19 04:35:24,981 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/publish.yml
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/lint.cfg
.kokoro/continuous/node10/samples-test.cfg
.kokoro/continuous/node10/system-test.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/continuous/node8/common.cfg
.kokoro/continuous/node8/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node10/docs.cfg
.kokoro/presubmit/node10/lint.cfg
.kokoro/presubmit/node10/samples-test.cfg
.kokoro/presubmit/node10/system-test.cfg
.kokoro/presubmit/node10/test.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/presubmit/node8/common.cfg
.kokoro/presubmit/node8/test.cfg
.kokoro/presubmit/windows/common.cfg
.kokoro/presubmit/windows/test.cfg
.kokoro/publish.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
renovate.json
samples/README.md
2020-05-19 04:35:25,109 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-05-19 04:35:25,109 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable
> npm run compile

npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> gts clean

version: 12
Removing build ...

> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> tsc -p . && cp -r proto* build/

[96msrc/instance.ts[0m:[93m890[0m:[93m20[0m - [91merror[0m[90m TS2339: [0mProperty 'gaxOptions' does not exist on type 'never'.
  The intersection 'GetTablesOptions & { parent: string; view: number; }' was reduced to 'never' because property 'view' has conflicting types in some constituents.

[7m890[0m     delete reqOpts.gaxOptions;
[7m   [0m [91m                   ~~~~~~~~~~[0m


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-19T11_35_39_533Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-19T11_35_39_553Z-debug.log
2020-05-19 04:35:39,567 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-05-19 04:35:39,568 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-05-19 04:35:39,617 autosynth [ERROR] > Synthesis failed
2020-05-19 04:35:39,617 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-19 04:35:39,630 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-05-19 04:35:39,639 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge/e68b2757-435d-4717-8786-35adc2df62d5).

Autosynth is still having trouble generating nodejs-bigtable. :sob:

Here's the output from running `synth.py`:

```
ol [DEBUG] > Generating code for: google/bigtable/v2.
DEBUG:synthtool:Generating code for: google/bigtable/v2.
2020-05-20 05:40:03,568 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpv94cab19.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpv94cab19.
2020-05-20 05:40:03,584 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:7bca0aa4c25f2cf6a07e1b6e1cd91a655c4cf010317b5b8fdde7e1aed0ba33d3
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-20 05:40:04,457 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.
DEBUG:synthtool:Generating code for: google/bigtable/admin/v2.
2020-05-20 05:40:05,781 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmptvbi3tnn.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmptvbi3tnn.
2020-05-20 05:40:05,813 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
2020-05-20 05:40:05,813 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
2020-05-20 05:40:05,814 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/publish.yml
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/lint.cfg
.kokoro/continuous/node10/samples-test.cfg
.kokoro/continuous/node10/system-test.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/continuous/node8/common.cfg
.kokoro/continuous/node8/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node10/docs.cfg
.kokoro/presubmit/node10/lint.cfg
.kokoro/presubmit/node10/samples-test.cfg
.kokoro/presubmit/node10/system-test.cfg
.kokoro/presubmit/node10/test.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/presubmit/node8/common.cfg
.kokoro/presubmit/node8/test.cfg
.kokoro/presubmit/windows/common.cfg
.kokoro/presubmit/windows/test.cfg
.kokoro/publish.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
renovate.json
samples/README.md
2020-05-20 05:40:05,954 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-05-20 05:40:05,955 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable
> npm run compile

npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> gts clean

version: 12
Removing build ...

> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> tsc -p . && cp -r proto* build/

[96msrc/instance.ts[0m:[93m890[0m:[93m20[0m - [91merror[0m[90m TS2339: [0mProperty 'gaxOptions' does not exist on type 'never'.
  The intersection 'GetTablesOptions & { parent: string; view: number; }' was reduced to 'never' because property 'view' has conflicting types in some constituents.

[7m890[0m     delete reqOpts.gaxOptions;
[7m   [0m [91m                   ~~~~~~~~~~[0m


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-20T12_40_21_724Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-20T12_40_21_741Z-debug.log
2020-05-20 05:40:21,755 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-05-20 05:40:21,757 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-05-20 05:40:21,795 autosynth [ERROR] > Synthesis failed
2020-05-20 05:40:21,795 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-20 05:40:21,808 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-05-20 05:40:21,816 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge/1d96130f-887f-4822-a400-bff75bdfb430).

Autosynth is still having trouble generating nodejs-bigtable. :sob:

Here's the output from running `synth.py`:

```
ol [DEBUG] > Generating code for: google/bigtable/v2.
DEBUG:synthtool:Generating code for: google/bigtable/v2.
2020-05-21 04:42:56,313 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpzl87_xtj.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpzl87_xtj.
2020-05-21 04:42:56,328 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:7bca0aa4c25f2cf6a07e1b6e1cd91a655c4cf010317b5b8fdde7e1aed0ba33d3
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-21 04:42:57,266 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.
DEBUG:synthtool:Generating code for: google/bigtable/admin/v2.
2020-05-21 04:42:58,636 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp8smlmxmd.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmp8smlmxmd.
2020-05-21 04:42:58,665 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
2020-05-21 04:42:58,666 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
2020-05-21 04:42:58,666 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/publish.yml
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/lint.cfg
.kokoro/continuous/node10/samples-test.cfg
.kokoro/continuous/node10/system-test.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/continuous/node8/common.cfg
.kokoro/continuous/node8/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node10/docs.cfg
.kokoro/presubmit/node10/lint.cfg
.kokoro/presubmit/node10/samples-test.cfg
.kokoro/presubmit/node10/system-test.cfg
.kokoro/presubmit/node10/test.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/presubmit/node8/common.cfg
.kokoro/presubmit/node8/test.cfg
.kokoro/presubmit/windows/common.cfg
.kokoro/presubmit/windows/test.cfg
.kokoro/publish.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
renovate.json
samples/README.md
2020-05-21 04:42:58,796 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-05-21 04:42:58,796 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable
> npm run compile

npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> gts clean

version: 12
Removing build ...

> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> tsc -p . && cp -r proto* build/

[96msrc/instance.ts[0m:[93m890[0m:[93m20[0m - [91merror[0m[90m TS2339: [0mProperty 'gaxOptions' does not exist on type 'never'.
  The intersection 'GetTablesOptions & { parent: string; view: number; }' was reduced to 'never' because property 'view' has conflicting types in some constituents.

[7m890[0m     delete reqOpts.gaxOptions;
[7m   [0m [91m                   ~~~~~~~~~~[0m


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-21T11_43_13_923Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-21T11_43_13_941Z-debug.log
2020-05-21 04:43:13,958 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-05-21 04:43:13,959 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-05-21 04:43:14,011 autosynth [ERROR] > Synthesis failed
2020-05-21 04:43:14,011 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-21 04:43:14,025 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-05-21 04:43:14,034 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge/2d09a9e0-e49f-4d2b-b5ea-2b554f1b32ec).

Autosynth is still having trouble generating nodejs-bigtable. :sob:

Here's the output from running `synth.py`:

```
ol [DEBUG] > Generating code for: google/bigtable/v2.
DEBUG:synthtool:Generating code for: google/bigtable/v2.
2020-05-22 05:33:10,690 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp6aph4qhj.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmp6aph4qhj.
2020-05-22 05:33:10,708 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:54099141c7227a5a0d343c99c879fa72b5f26d6ad7f0d168159b66ca0fd67e8a
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-22 05:33:11,613 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.
DEBUG:synthtool:Generating code for: google/bigtable/admin/v2.
2020-05-22 05:33:13,041 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmph9j7hi3z.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmph9j7hi3z.
2020-05-22 05:33:13,077 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
2020-05-22 05:33:13,078 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
2020-05-22 05:33:13,078 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/publish.yml
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/lint.cfg
.kokoro/continuous/node10/samples-test.cfg
.kokoro/continuous/node10/system-test.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/continuous/node8/common.cfg
.kokoro/continuous/node8/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node10/docs.cfg
.kokoro/presubmit/node10/lint.cfg
.kokoro/presubmit/node10/samples-test.cfg
.kokoro/presubmit/node10/system-test.cfg
.kokoro/presubmit/node10/test.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/presubmit/node8/common.cfg
.kokoro/presubmit/node8/test.cfg
.kokoro/presubmit/windows/common.cfg
.kokoro/presubmit/windows/test.cfg
.kokoro/publish.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
renovate.json
samples/README.md
2020-05-22 05:33:13,228 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-05-22 05:33:13,228 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable
> npm run compile

npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> gts clean

version: 12
Removing build ...

> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> tsc -p . && cp -r proto* build/

[96msrc/instance.ts[0m:[93m890[0m:[93m20[0m - [91merror[0m[90m TS2339: [0mProperty 'gaxOptions' does not exist on type 'never'.
  The intersection 'GetTablesOptions & { parent: string; view: number; }' was reduced to 'never' because property 'view' has conflicting types in some constituents.

[7m890[0m     delete reqOpts.gaxOptions;
[7m   [0m [91m                   ~~~~~~~~~~[0m


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-22T12_33_29_532Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-22T12_33_29_552Z-debug.log
2020-05-22 05:33:29,567 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-05-22 05:33:29,569 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-05-22 05:33:29,613 autosynth [ERROR] > Synthesis failed
2020-05-22 05:33:29,613 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-22 05:33:29,626 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-05-22 05:33:29,635 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge/8aeddfbb-7b58-4fc8-9895-2ada12afbca7).

Autosynth is still having trouble generating nodejs-bigtable. :sob:

Here's the output from running `synth.py`:

```
ol [DEBUG] > Generating code for: google/bigtable/v2.
DEBUG:synthtool:Generating code for: google/bigtable/v2.
2020-05-23 11:41:50,790 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpfe0h2v5_.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpfe0h2v5_.
2020-05-23 11:41:50,807 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:54099141c7227a5a0d343c99c879fa72b5f26d6ad7f0d168159b66ca0fd67e8a
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-23 11:41:51,809 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.
DEBUG:synthtool:Generating code for: google/bigtable/admin/v2.
2020-05-23 11:41:53,361 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpqt5iu2v3.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpqt5iu2v3.
2020-05-23 11:41:53,392 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
2020-05-23 11:41:53,392 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
2020-05-23 11:41:53,392 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/publish.yml
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/lint.cfg
.kokoro/continuous/node10/samples-test.cfg
.kokoro/continuous/node10/system-test.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/continuous/node8/common.cfg
.kokoro/continuous/node8/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node10/docs.cfg
.kokoro/presubmit/node10/lint.cfg
.kokoro/presubmit/node10/samples-test.cfg
.kokoro/presubmit/node10/system-test.cfg
.kokoro/presubmit/node10/test.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/presubmit/node8/common.cfg
.kokoro/presubmit/node8/test.cfg
.kokoro/presubmit/windows/common.cfg
.kokoro/presubmit/windows/test.cfg
.kokoro/publish.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
renovate.json
samples/README.md
2020-05-23 11:41:53,534 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-05-23 11:41:53,535 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable
> npm run compile

npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> gts clean

version: 12
Removing build ...

> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> tsc -p . && cp -r proto* build/

[96msrc/instance.ts[0m:[93m890[0m:[93m20[0m - [91merror[0m[90m TS2339: [0mProperty 'gaxOptions' does not exist on type 'never'.
  The intersection 'GetTablesOptions & { parent: string; view: number; }' was reduced to 'never' because property 'view' has conflicting types in some constituents.

[7m890[0m     delete reqOpts.gaxOptions;
[7m   [0m [91m                   ~~~~~~~~~~[0m


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-23T18_42_10_290Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-23T18_42_10_309Z-debug.log
2020-05-23 11:42:10,326 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-05-23 11:42:10,327 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-05-23 11:42:10,398 autosynth [ERROR] > Synthesis failed
2020-05-23 11:42:10,399 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-23 11:42:10,413 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-05-23 11:42:10,424 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge/55b41abd-49e7-468a-beba-a8683b58c54c).

Autosynth is still having trouble generating nodejs-bigtable. :sob:

Here's the output from running `synth.py`:

```
ol [DEBUG] > Generating code for: google/bigtable/v2.
DEBUG:synthtool:Generating code for: google/bigtable/v2.
2020-05-24 09:40:13,852 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp64sa1z4b.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmp64sa1z4b.
2020-05-24 09:40:13,869 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:54099141c7227a5a0d343c99c879fa72b5f26d6ad7f0d168159b66ca0fd67e8a
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-24 09:40:14,754 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.
DEBUG:synthtool:Generating code for: google/bigtable/admin/v2.
2020-05-24 09:40:16,143 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpb7pjqf26.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpb7pjqf26.
2020-05-24 09:40:16,174 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
2020-05-24 09:40:16,175 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
2020-05-24 09:40:16,175 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/publish.yml
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/lint.cfg
.kokoro/continuous/node10/samples-test.cfg
.kokoro/continuous/node10/system-test.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/continuous/node8/common.cfg
.kokoro/continuous/node8/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node10/docs.cfg
.kokoro/presubmit/node10/lint.cfg
.kokoro/presubmit/node10/samples-test.cfg
.kokoro/presubmit/node10/system-test.cfg
.kokoro/presubmit/node10/test.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/presubmit/node8/common.cfg
.kokoro/presubmit/node8/test.cfg
.kokoro/presubmit/windows/common.cfg
.kokoro/presubmit/windows/test.cfg
.kokoro/publish.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
renovate.json
samples/README.md
2020-05-24 09:40:16,312 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-05-24 09:40:16,312 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable
> npm run compile

npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> gts clean

version: 12
Removing build ...

> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> tsc -p . && cp -r proto* build/

[96msrc/instance.ts[0m:[93m890[0m:[93m20[0m - [91merror[0m[90m TS2339: [0mProperty 'gaxOptions' does not exist on type 'never'.
  The intersection 'GetTablesOptions & { parent: string; view: number; }' was reduced to 'never' because property 'view' has conflicting types in some constituents.

[7m890[0m     delete reqOpts.gaxOptions;
[7m   [0m [91m                   ~~~~~~~~~~[0m


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-24T16_40_32_180Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-24T16_40_32_198Z-debug.log
2020-05-24 09:40:32,212 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-05-24 09:40:32,214 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-05-24 09:40:32,257 autosynth [ERROR] > Synthesis failed
2020-05-24 09:40:32,257 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-24 09:40:32,270 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-05-24 09:40:32,279 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge/1b789978-f73a-46c5-b32f-b0f5a9fa15cc).

Autosynth is still having trouble generating nodejs-bigtable. :sob:

Here's the output from running `synth.py`:

```
ol [DEBUG] > Generating code for: google/bigtable/v2.
DEBUG:synthtool:Generating code for: google/bigtable/v2.
2020-05-25 07:43:21,546 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp0bbs26y9.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmp0bbs26y9.
2020-05-25 07:43:21,563 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:54099141c7227a5a0d343c99c879fa72b5f26d6ad7f0d168159b66ca0fd67e8a
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-25 07:43:22,463 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.
DEBUG:synthtool:Generating code for: google/bigtable/admin/v2.
2020-05-25 07:43:23,778 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpfofrfir6.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpfofrfir6.
2020-05-25 07:43:23,810 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
2020-05-25 07:43:23,810 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
2020-05-25 07:43:23,810 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/publish.yml
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/lint.cfg
.kokoro/continuous/node10/samples-test.cfg
.kokoro/continuous/node10/system-test.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/continuous/node8/common.cfg
.kokoro/continuous/node8/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node10/docs.cfg
.kokoro/presubmit/node10/lint.cfg
.kokoro/presubmit/node10/samples-test.cfg
.kokoro/presubmit/node10/system-test.cfg
.kokoro/presubmit/node10/test.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/presubmit/node8/common.cfg
.kokoro/presubmit/node8/test.cfg
.kokoro/presubmit/windows/common.cfg
.kokoro/presubmit/windows/test.cfg
.kokoro/publish.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
renovate.json
samples/README.md
2020-05-25 07:43:23,956 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-05-25 07:43:23,957 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable
> npm run compile

npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> gts clean

version: 12
Removing build ...

> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> tsc -p . && cp -r proto* build/

[96msrc/instance.ts[0m:[93m890[0m:[93m20[0m - [91merror[0m[90m TS2339: [0mProperty 'gaxOptions' does not exist on type 'never'.
  The intersection 'GetTablesOptions & { parent: string; view: number; }' was reduced to 'never' because property 'view' has conflicting types in some constituents.

[7m890[0m     delete reqOpts.gaxOptions;
[7m   [0m [91m                   ~~~~~~~~~~[0m


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-25T14_43_40_372Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-25T14_43_40_393Z-debug.log
2020-05-25 07:43:40,407 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-05-25 07:43:40,409 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-05-25 07:43:40,447 autosynth [ERROR] > Synthesis failed
2020-05-25 07:43:40,448 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-25 07:43:40,461 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-05-25 07:43:40,469 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge/ba51bb8e-983e-4ef5-846f-55ea9fa4e79c).

Autosynth is still having trouble generating nodejs-bigtable. :sob:

Here's the output from running `synth.py`:

```
ol [DEBUG] > Generating code for: google/bigtable/v2.
DEBUG:synthtool:Generating code for: google/bigtable/v2.
2020-05-26 06:53:58,211 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpltxd3fl7.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpltxd3fl7.
2020-05-26 06:53:58,226 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest
DEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest
latest: Pulling from gapic-images/gapic-generator-typescript
Digest: sha256:54099141c7227a5a0d343c99c879fa72b5f26d6ad7f0d168159b66ca0fd67e8a
Status: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest
2020-05-26 06:53:59,092 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.
DEBUG:synthtool:Generating code for: google/bigtable/admin/v2.
2020-05-26 06:54:00,600 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmprdhzrawm.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmprdhzrawm.
2020-05-26 06:54:00,632 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.ts.
2020-05-26 06:54:00,632 synthtool [INFO] > Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
INFO:synthtool:Replaced 'BigtableClient' in system-test/fixtures/sample/src/index.js.
2020-05-26 06:54:00,632 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
.eslintignore
.eslintrc.json
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/publish.yml
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/lint.cfg
.kokoro/continuous/node10/samples-test.cfg
.kokoro/continuous/node10/system-test.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/continuous/node8/common.cfg
.kokoro/continuous/node8/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node10/docs.cfg
.kokoro/presubmit/node10/lint.cfg
.kokoro/presubmit/node10/samples-test.cfg
.kokoro/presubmit/node10/system-test.cfg
.kokoro/presubmit/node10/test.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/presubmit/node8/common.cfg
.kokoro/presubmit/node8/test.cfg
.kokoro/presubmit/windows/common.cfg
.kokoro/presubmit/windows/test.cfg
.kokoro/publish.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
renovate.json
samples/README.md
2020-05-26 06:54:00,775 synthtool [DEBUG] > Post-processing GAPIC library...
DEBUG:synthtool:Post-processing GAPIC library...
2020-05-26 06:54:00,775 synthtool [DEBUG] > Installing dependencies...
DEBUG:synthtool:Installing dependencies...
npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable
> npm run compile

npm WARN npm npm does not support Node.js v12.16.3
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> gts clean

version: 12
Removing build ...

> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable
> tsc -p . && cp -r proto* build/

[96msrc/instance.ts[0m:[93m890[0m:[93m20[0m - [91merror[0m[90m TS2339: [0mProperty 'gaxOptions' does not exist on type 'never'.
  The intersection 'GetTablesOptions & { parent: string; view: number; }' was reduced to 'never' because property 'view' has conflicting types in some constituents.

[7m890[0m     delete reqOpts.gaxOptions;
[7m   [0m [91m                   ~~~~~~~~~~[0m


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-26T13_54_16_768Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-05-26T13_54_16_786Z-debug.log
2020-05-26 06:54:16,801 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-05-26 06:54:16,803 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-05-26 06:54:16,842 autosynth [ERROR] > Synthesis failed
2020-05-26 06:54:16,842 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 998dc4b fix: Point to team in correct org (#729)
2020-05-26 06:54:16,856 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-05-26 06:54:16,864 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge/0344a2cb-00c6-4ee5-a325-b78b8a82b7b8).

Autosynth is still having trouble generating nodejs-bigtable. :sob:

Here's the output from running `synth.py`:

```
b'ol [DEBUG] > Generating code for: google/bigtable/v2.\nDEBUG:synthtool:Generating code for: google/bigtable/v2.\n2020-05-27 04:35:41,904 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpyz361fes.\nSUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpyz361fes.\n2020-05-27 04:35:41,919 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest\nDEBUG:synthtool:Pulling Docker image: gapic-generator-typescript:latest\nlatest: Pulling from gapic-images/gapic-generator-typescript\nDigest: sha256:e1d667bda503828c8ebd45af3554d8b5b4c45a3753d05b60a9b47c03579f1bdf\nStatus: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest\n2020-05-27 04:35:42,797 synthtool [DEBUG] > Generating code for: google/bigtable/admin/v2.\nDEBUG:synthtool:Generating code for: google/bigtable/admin/v2.\n2020-05-27 04:35:44,112 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpqld7csez.\nSUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpqld7csez.\n2020-05-27 04:35:44,141 synthtool [INFO] > Replaced \'BigtableClient\' in system-test/fixtures/sample/src/index.ts.\nINFO:synthtool:Replaced \'BigtableClient\' in system-test/fixtures/sample/src/index.ts.\n2020-05-27 04:35:44,142 synthtool [INFO] > Replaced \'BigtableClient\' in system-test/fixtures/sample/src/index.js.\nINFO:synthtool:Replaced \'BigtableClient\' in system-test/fixtures/sample/src/index.js.\n2020-05-27 04:35:44,142 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool\nDEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool\n.eslintignore\n.eslintrc.json\n.github/ISSUE_TEMPLATE/bug_report.md\n.github/ISSUE_TEMPLATE/feature_request.md\n.github/ISSUE_TEMPLATE/support_request.md\n.github/PULL_REQUEST_TEMPLATE.md\n.github/publish.yml\n.github/release-please.yml\n.github/workflows/ci.yaml\n.kokoro/common.cfg\n.kokoro/continuous/node10/common.cfg\n.kokoro/continuous/node10/docs.cfg\n.kokoro/continuous/node10/lint.cfg\n.kokoro/continuous/node10/samples-test.cfg\n.kokoro/continuous/node10/system-test.cfg\n.kokoro/continuous/node10/test.cfg\n.kokoro/continuous/node12/common.cfg\n.kokoro/continuous/node12/test.cfg\n.kokoro/continuous/node8/common.cfg\n.kokoro/continuous/node8/test.cfg\n.kokoro/docs.sh\n.kokoro/lint.sh\n.kokoro/presubmit/node10/common.cfg\n.kokoro/presubmit/node10/docs.cfg\n.kokoro/presubmit/node10/lint.cfg\n.kokoro/presubmit/node10/samples-test.cfg\n.kokoro/presubmit/node10/system-test.cfg\n.kokoro/presubmit/node10/test.cfg\n.kokoro/presubmit/node12/common.cfg\n.kokoro/presubmit/node12/test.cfg\n.kokoro/presubmit/node8/common.cfg\n.kokoro/presubmit/node8/test.cfg\n.kokoro/presubmit/windows/common.cfg\n.kokoro/presubmit/windows/test.cfg\n.kokoro/publish.sh\n.kokoro/release/docs.cfg\n.kokoro/release/docs.sh\n.kokoro/release/publish.cfg\n.kokoro/samples-test.sh\n.kokoro/system-test.sh\n.kokoro/test.bat\n.kokoro/test.sh\n.kokoro/trampoline.sh\n.mocharc.js\n.nycrc\n.prettierignore\n.prettierrc.js\nCODE_OF_CONDUCT.md\nCONTRIBUTING.md\nLICENSE\nREADME.md\nrenovate.json\nsamples/README.md\n2020-05-27 04:35:44,268 synthtool [DEBUG] > Post-processing GAPIC library...\nDEBUG:synthtool:Post-processing GAPIC library...\n2020-05-27 04:35:44,269 synthtool [DEBUG] > Installing dependencies...\nDEBUG:synthtool:Installing dependencies...\nnpm WARN npm npm does not support Node.js v12.17.0\nnpm WARN npm You should probably upgrade to a newer version of node as we\nnpm WARN npm can\'t make any promises that npm will work with this version.\nnpm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.\nnpm WARN npm You can find the latest version at https://nodejs.org/\nnpm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.\n\n> @google-cloud/bigtable@2.3.2 prepare /home/kbuilder/.cache/synthtool/nodejs-bigtable\n> npm run compile\n\nnpm WARN npm npm does not support Node.js v12.17.0\nnpm WARN npm You should probably upgrade to a newer version of node as we\nnpm WARN npm can\'t make any promises that npm will work with this version.\nnpm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11.\nnpm WARN npm You can find the latest version at https://nodejs.org/\n\n> @google-cloud/bigtable@2.3.2 precompile /home/kbuilder/.cache/synthtool/nodejs-bigtable\n> gts clean\n\nversion: 12\nRemoving build ...\n\n> @google-cloud/bigtable@2.3.2 compile /home/kbuilder/.cache/synthtool/nodejs-bigtable\n> tsc -p . && cp -r proto* build/\n\n\x1b[96msrc/instance.ts\x1b[0m:\x1b[93m890\x1b[0m:\x1b[93m20\x1b[0m - \x1b[91merror\x1b[0m\x1b[90m TS2339: \x1b[0mProperty \'gaxOptions\' does not exist on type \'never\'.\n  The intersection \'GetTablesOptions & { parent: string; view: number; }\' was reduced to \'never\' because property \'view\' has conflicting types in some constituents.\n\n\x1b[7m890\x1b[0m     delete reqOpts.gaxOptions;\n\x1b[7m   \x1b[0m \x1b[91m                   ~~~~~~~~~~\x1b[0m\n\n\nFound 1 error.\n\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 1\nnpm ERR! @google-cloud/bigtable@2.3.2 compile: `tsc -p . && cp -r proto* build/`\nnpm ERR! Exit status 1\nnpm ERR! \nnpm ERR! Failed at the @google-cloud/bigtable@2.3.2 compile script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /home/kbuilder/.npm/_logs/2020-05-27T11_35_58_802Z-debug.log\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 1\nnpm ERR! @google-cloud/bigtable@2.3.2 prepare: `npm run compile`\nnpm ERR! Exit status 1\nnpm ERR! \nnpm ERR! Failed at the @google-cloud/bigtable@2.3.2 prepare script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /home/kbuilder/.npm/_logs/2020-05-27T11_35_58_818Z-debug.log\n2020-05-27 04:35:58,832 synthtool [ERROR] > Failed executing npm install:\n\nNone\nERROR:synthtool:Failed executing npm install:\n\nNone\n2020-05-27 04:35:58,833 synthtool [DEBUG] > Wrote metadata to synth.metadata.\nDEBUG:synthtool:Wrote metadata to synth.metadata.\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__\n    return self.main(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main\n    rv = self.invoke(ctx)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke\n    return callback(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main\n    spec.loader.exec_module(synth_module)  # type: ignore\n  File "<frozen importlib._bootstrap_external>", line 678, in exec_module\n  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed\n  File "/home/kbuilder/.cache/synthtool/nodejs-bigtable/synth.py", line 59, in <module>\n    node.postprocess_gapic_library()\n  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library\n    install(hide_output=hide_output)\n  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install\n    shell.run(["npm", "install"], hide_output=hide_output)\n  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run\n    raise exc\n  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run\n    encoding="utf-8",\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run\n    output=stdout, stderr=stderr)\nsubprocess.CalledProcessError: Command \'[\'npm\', \'install\']\' returned non-zero exit status 1.\n2020-05-27 04:35:58,880 autosynth [ERROR] > Synthesis failed\n2020-05-27 04:35:58,880 autosynth [DEBUG] > Running: git reset --hard HEAD\nHEAD is now at 998dc4b fix: Point to team in correct org (#729)\n2020-05-27 04:35:58,894 autosynth [DEBUG] > Running: git checkout autosynth\nSwitched to branch \'autosynth\'\n2020-05-27 04:35:58,903 autosynth [DEBUG] > Running: git clean -fdx\nRemoving __pycache__/\nRemoving node_modules/\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main\n    return _inner_main(temp_dir)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main\n    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop\n    synthesize_inner_loop(toolbox, synthesizer)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop\n    synthesizer, len(toolbox.versions) - 1\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch\n    synthesizer.synthesize(synth_log_path, self.environ)\n  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize\n    synth_proc.check_returncode()  # Raise an exception.\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode\n    self.stderr)\nsubprocess.CalledProcessError: Command \'[\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\' returned non-zero exit status 1.\n'
```

Google internal developers can see the full log [here](http://sponge/51e18e2a-b562-4e19-987f-9aedba25285d).

--- a/src/instance.ts
+++ b/src/instance.ts
@@ -887,7 +887,7 @@ Please use the format 'my-instance' or '${bigtable.projectName}/instances/my-ins
       view: Table.VIEWS[options.view || 'unspecified'],
     });
 
-    delete reqOpts.gaxOptions;
+    delete (reqOpts as GetTablesOptions).gaxOptions;
 
     this.bigtable.request<Table[]>(
       {

